CHANGELOG.md

```diff
@@ -1,21 +1,5 @@
 # History of changes
 
-## Version X.Y.Z
-
-### API Changes
-
-* `spyderlib.dicteditor.DictEditor` has been renamed to `spyderlib.widgets.editors.collectionseditor.CollectionsEditor`
-* Editor widgets were moved from `spyderlib.widgets` to `spyderlib.widgets.editors`:
-    * `spyderlib.widgets.editors.arrayeditor`
-    * `spyderlib.widgets.editors.collectionseditor`
-    * `spyderlib.widgets.editors.objecteditor`
-    * `spyderlib.widgets.editors.texteditor`
-    * `spyderlib.widgets.editors.dataframeeditor`
-

```

Comment:
I'm keeping a record of new things for Spyder 3.0 in our wiki, not here

https://github.com/spyder-ide/spyder/wiki/Beta-version-changelog